src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -129,6 +129,10 @@ object Config {
 
     val maxQueuedBlockNumberAhead: Int
     val maxQueuedBlockNumberBehind: Int
+
+    // HardCoded values based on parity client
+    val maxNewBlockHashAge = 20
+    val maxNewHashes = 64
```

Comment:
Minor detail: Maybe we could make this values be obtained from the `application.conf` file?